collector_k8s/scrape_configs_ds.yaml

```diff
@@ -0,0 +1,19 @@
+- job_name: apps
```

Comment:
Do we also need to make a job here for each DS collector to scrape itself?

Reply:
Ah i see we add the annotations below in the pod annotation spec, does that work as expected?

Reply:
Yeah, that's right and it does work.